### DISCUSSION TITLE
Unable to execute command in powershell

### DISCUSSION THREAD

**Discussion by SkyLineMMM** [Markemp/Cryengine-Converter#162] [Technical]
Hello everyone, when I run the command:

cgf-converter.exe .\bu_d_1fstair01_e_01a.cgf -objectdir -dae "g:\1\AION" 

gives an error message:

cgf-converter.exe: The term 'cgf-converter.exe' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Suggestion [3,General]: The command cgf-converter.exe was not found, but does exist in the current location. PowerShell does not load commands from the current location by default. If you trust this command, instead type: ".\cgf-converter.exe". See "get-help about_Command_Precedence" for more details.

![image](https://github.com/Markemp/Cryengine-Converter/assets/94620658/25e815f0-5396-48fd-a399-945800be5db8)

Anyone can help with this problem?

**Comment by Markemp** [comment]
That generally means that wherever you put the .exe file, it's not in your path.  You can either add that directory to your path, or use the fully qualified path to help the terminal find the exe.  For example, if you have it in `d:\scripts\` like I do, you can use `d:\scripts\cgf-converter.exe` in place of just `cgf-converter.exe`.

Also, if it's in the current directory like it appears in your error message (which I didn't fully read the first time, because damn you Chimay!), you can type `.\cgf-converter.exe` instead. 😊

**Reply by SkyLineMMM** [reply]
I did as you suggested above, the script script is executed without errors, but the converted file is not created.

![image](https://github.com/Markemp/Cryengine-Converter/assets/94620658/328ec044-eb0f-4b27-9731-9a68c6bc47c7)

**Comment by Dizcordum** [comment]
![gif73](https://github.com/Markemp/Cryengine-Converter/assets/68729538/3c60c343-cd03-4347-8350-0a24146ca55d)
Open cmd like this in folder where you planning to use converter

**Reply by SkyLineMMM** [reply]
Yes, I tried to do that too
